README.md

```diff
@@ -88,75 +89,72 @@ Snippets, on the other hand, allow you to quickly insert small examples of Affin
 
 Available snippets:
 
-| Name | Description | Supported Languages |
-|---|---|---|
-| affinidi-create-issuance | Create bulk issuance entity (using fetch) | Typescript and Javascript |
-| affinidi-create-offer | Create & send bulk issuance offer (using fetch) | Typescript and Javascript |
+| Name                     | Description                                     | Supported Languages       |
+| ------------------------ | ----------------------------------------------- | ------------------------- |
+| affinidi-create-issuance | Create bulk issuance entity (using fetch)       | Typescript and Javascript |
+| affinidi-create-offer    | Create & send bulk issuance offer (using fetch) | Typescript and Javascript |
 
 > For instructions on how to use snippets please refer to [Visual Studio Code snippets documentation](https://code.visualstudio.com/docs/editor/userdefinedsnippets).
 
 ## Reference app
 
-If you want an end-to-end example of an application, that uses Affinidi APIs, you can generate a reference app directly from the extension.  
-Reference app includes such features as: issuing a credential, claiming & storing it in your wallet, verifying it.  
+If you want an end-to-end example of an application, that uses Affinidi APIs, you can generate 2 types reference apps directly from the extension.  
+Certification and Verification app includes such features as: issuing a credential, claiming & storing it in your wallet, verifying it.
 
-Just click "Certification & Verification" button in "App Generators" category:
+Portable Reputation app includes such features as: connecting to different data sources to construct a portable, user-owned and -managed reputation that can give their user access to specific experiences, events or memberships..
```

Comment:
"The Portable Reputation app allows the builder to connect to different data sources to construct a portable, user-owned and -managed reputation app that provides access to specific experiences, events or memberships."